pyrsa/rdm/calc.py

```diff
@@ -130,27 +141,43 @@ def calc_rdm_mahalanobis(dataset, descriptor=None, noise=None):
         pyrsa.rdm.rdms.RDMs: RDMs object with the one RDM
 
     """
-    measurements, desc, descriptor = _parse_input(dataset, descriptor)
-    noise = _check_noise(noise, dataset.n_channel)
-    diff = _calc_pairwise_differences(measurements)
-    diff2 = (noise @ diff.T).T
-    rdm = np.einsum('ij,ij->i', diff, diff2) / measurements.shape[1]
-    rdm = RDMs(dissimilarities=np.array([rdm]),
-               dissimilarity_measure='Mahalanobis',
-               descriptors=dataset.descriptors)
-    rdm.pattern_descriptors[descriptor] = desc
-    rdm.descriptors['noise'] = noise
+    if noise is None:
+        rdm = calc_rdm_euclid(dataset, descriptor)
```

Comment:
Add warning that a precision matrix needs to be provided and that you automatically revert to Euclidian distance estimates otherwise

Reply:
I put a comment on that into the help strings. I don't want to raise a warning as this can be completely intentional, for example to calculate a cross-validated Euclidean distance.